bigchaindb/core.py

```diff
@@ -485,3 +485,25 @@ def get_unvoted_blocks(self):
             unvoted.pop(0)
 
         return unvoted
+
+    def block_election_status(self, block):
+        """
+        Tallies the votes on a block, and returns the status: valid, invalid, or undecided
+        """
+        n_voters = len(block['block']['voters'])
+
+        vote_list = [vote['vote']['is_block_valid']
+                     for vote in block['block']['votes']
+                     if self.consensus.verify_vote_signature(vote)]
+
+        # validate votes here
+
+        n_invalid_votes = vote_list.count(False)
+        n_valid_votes = vote_list.count(True)
+
+        if n_invalid_votes >= int(n_voters/2):
+            return 'invalid'
```

Comment:
Usually constants are at a [module level](https://www.python.org/dev/peps/pep-0008/#constants) but in this case I think it's fine to keep them on a class level.